utilities/save_data_functions.py

```diff
@@ -1,4 +1,4 @@
-def save_temp_field_chamber_res_data():
+def save_temp_field_chamber_res_data(bridge):
```

Comment:
Please make this change backwards-compatible so we don't break any other scripts that rely on this utility. Just default the new parameter to 1.